pkg/noun/v1/allocate.h

```diff
@@ -0,0 +1,61 @@
+#ifndef U3_ALLOCATE_V1_H
+#define U3_ALLOCATE_V1_H
+
+#include "pkg/noun/allocate.h"
+#include "pkg/noun/v2/allocate.h"
+
+  /**  Aliases.
+  **/
+#     define u3R_v1         u3R_v2
+#     define u3a_v1_botox   u3a_v2_botox
+#     define u3a_v1_box     u3a_v2_box
+#     define u3a_v1_cell    u3a_v2_cell
+#     define u3a_v1_free    u3a_v2_free
+#     define u3a_v1_into    u3a_v2_into
+#     define u3a_v1_is_cat  u3a_v2_is_cat
+#     define u3a_v1_is_pom  u3a_is_pom
+#     define u3a_v1_wfree   u3a_v2_wfree
```

Comment:
The deallocation functions will need to be pulled in from the commit prior to pointer compressoin.